packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/java/enrollmentintegrationsteps.md

```diff
@@ -0,0 +1,98 @@
+### 1: Build a sign-in page on the client
+
+Build a sign-in page that capture's the user's name and password, as shown in the following example.
+
+![Basic sign-in dialog](/img/authenticators/java-authenticators-signinform.png)
+
+### 2: Authenticate the user credentials
+
+After a user initiates the sign-in flow by entering their username and password and then clicking **Sign In**, add the following code:
+
+```java
+// create an IDX Authentication Wrapper (can be an application scoped singleton)
+IDXAuthenticationWrapper idxAuthenticationWrapper = new IDXAuthenticationWrapper();
+
+// begin transaction
+// The proceedContext needs to be persisted between interactions (e.g. stored in a HttpSession for web apps)
+ProceedContext proceedContext = idxAuthenticationWrapper.begin().getProceedContext();
+
+// set the user's credentials
+AuthenticationOptions authenticationOptions = new AuthenticationOptions(username, password);
+
+// start authentication flow
+AuthenticationResponse authenticationResponse = idxAuthenticationWrapper.authenticate(authenticationOptions, proceedContext);
+```
+
+### 3: Handle the response from the sign-in flow
+
+The `AuthenticationResponse` contains the current state of the authentication process. It could be a `SUCCESS` or may indicate more information is needed to complete the flow, for example, require an additional factor.
```

Comment:
```suggestion
The `AuthenticationResponse` contains the current state of the authentication process. It could be a `SUCCESS` or may indicate that more information is needed to complete the flow, for example, require an additional factor.
```